iree/compiler/Dialect/HAL/Target/ROCM/ROCMTarget.cpp

```diff
@@ -144,12 +146,18 @@ class ROCMTargetBackend final : public TargetBackend {
     iree_compiler::FlatbufferBuilder builder;
     iree_ROCMExecutableDef_start_as_root(builder);
 
+    // Link module to Device Library
+#ifdef LINK_ROCM_ON
```

Comment:
I would just use a `llvm::cl::opt<bool>` as it doesn't need to be a compile time choice. This way it makes the build simpler and allow to decide at runtime whether to link or not with command line.

Reply:
Done :)